### ISSUE TITLE
More suggestions

### ISSUE THREAD

**Issue by ATHDevelopers** [Henry-Hiles/QuadraticBot2.0#12]
Suggestions - Things to add
+ An auto reroll feature
+ Bonus role entries
+ Help menu
+ Add a color option in /config for the giveaway embed color
+ Make the giveaway button reply in embed message
+ A status command to view the bot status- Total server/users/giveaway hosted/active giveaways
+ A /edit command to edit giveaways 
+ More requirements

**Comment by Henry-Hiles** [comment]
I am planning on adding a /help command, and maybe /edit. The status idea is cool but low-priority. I can easily add a color, i will do that. Bonus role entries are a little harder, but I can do that in the future. I will put that in an embed, good idea. Not sure what you mran by auto reroll, could you elaborate? And for more requirements: not right now, but maybe in the future. These will take a while, if you want to speed up the process/help it would be great if you could make a PR. Thx for the organized msg.

**Comment by Henry-Hiles** [comment]
I can add the coloe feature tomorrow, and thx for reminding me about the changelog, will update that tomorrow as well.

**Comment by Henry-Hiles** [comment]
Ok i will also do that.

**Comment by Henry-Hiles** [comment]
Hmm OK.
i actually wont be able to do this for 2 days, i am a bit busy.

**Comment by Henry-Hiles** [comment]
Really? let me check settings quickly

**Comment by Henry-Hiles** [comment]
You should be able to as far as i can see, what is the issue?

**Comment by Henry-Hiles** [comment]
This explains the process: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by Henry-Hiles** [comment]
Specifically https://docs.github.com/en/articles/creating-a-pull-request-from-a-fork

**Comment by Henry-Hiles** [comment]
> More suggestions - Things to add
> 
>     * Only users with the permission `MANAGE_SERVER` can create giveaways
> 
>     * Auto re-roll command /auto-reroll `minutes`: Max limit 48h (auto reroll a giveaway)

That first one is built into discord, and what is the use fir the second may i ask?

**Comment by Henry-Hiles** [comment]
> A /giveaway-cancle `giveaway:` command would be great to cancle giveaways

You can just delete the message for that. However, if you still think its a good feature, i can do that super quickly.

**Comment by Henry-Hiles** [comment]
Not sure, maybe tomorrow, if not it may br a while, going on holiday.

**Comment by ANNACRAFTE** [comment]
He keep saying he is gonna do it and he dont smh

**Comment by Henry-Hiles** [comment]
I said i might be able to. If you want a feature right away make a PR

**Comment by ANNACRAFTE** [comment]
> You know what never mind i wont be avaliable anymore Till next year march 😢

Too bad for you Akari

**Comment by Henry-Hiles** [comment]
I should be able to do some of these after my holiday.

**Comment by Henry-Hiles** [comment]
> 

Yes, I have school, I did the az-900 exam, and i need exercise. But after my holiday I should have time :)

**Comment by Henry-Hiles** [comment]
On it rn

**Comment by Henry-Hiles** [comment]
Wdym?

**Comment by Henry-Hiles** [comment]
> Are you avaliable at this time?

No, as I said, I am on holiday.

**Comment by Henry-Hiles** [comment]
> I found a bug btw

Please open a seperate issue for that, and br motr specific please :)

**Comment by Henry-Hiles** [comment]
Just got back, should be able to add these soon! Unfortunatly I have covid, so it may take a while but i will work on it. Also, would an official discord server for support be good?

**Comment by Henry-Hiles** [comment]
Feeling pretty rough with covid rn, I have fixed the bug, but I won't be able to do these features until later. However, in the meantime i will make a support server.

**Comment by Henry-Hiles** [comment]
Hey @Akari0001, could I have your discord username to DM you to give you an early invite link?

**Comment by Henry-Hiles** [comment]
OK, sent you a friend request, I am QuadRadical.

**Comment by Henry-Hiles** [comment]
Oh wait, you aren't accepting friend requests.

**Comment by Henry-Hiles** [comment]
You can add me if you want. `QuadRadical#9516`

**Comment by Henry-Hiles** [comment]
> Add a `server` option in /giveaway
> 
> The server option is where we put a server invite, it will show in the requirements field as
> 
> > Must be a member of {servername}

This would be display only though?

**Comment by Henry-Hiles** [comment]
> A /giveaway-cancle `giveaway:` command would be great to cancle giveaways

would a context menu work? I think i will do that.

**Comment by Henry-Hiles** [comment]
I have used a context menu.

**Comment by Henry-Hiles** [comment]
> Wait i got another one If users pick more than 1 winners the winner field in the embed will show as _WINNERS_ but if the user pick 1 winner it will how as _WINNER_

Where would this change be?

**Comment by Henry-Hiles** [comment]
> Suggestions - Things to add
> 
>     * An auto reroll feature
> 
>     * Bonus role entries
> 
>     * Help menu
> 
>     * Add a color option in /config for the giveaway embed color
> 
>     * Make the giveaway button reply in embed message
> 
>     * A status command to view the bot status- Total server/users/giveaway hosted/active giveaways
> 
>     * A /edit command to edit giveaways
> 
>     * More requirements

I have implemented `/help`.
I don't plan to implement auto reroll.
I think the button reply is better as a message.
I will at some point add a custom requirements option.
I will add `/status` sometime.
I will add bonus role entries at some point, but it may be a while.
I'm not sure if i will add `/edit`.

**Comment by Henry-Hiles** [comment]
> > 
> 
> What do you mean

Where does it say winners instead of winner?
